CHANGELOG.md

```diff
@@ -5,7 +5,23 @@ All notable changes to htmltools for Python will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [UNRELEASED]
+## [Unreleased] YYYY-MM-DD
+
+### Breaking changes
+
+* `HTML` no longer inherits from `str`. It now inherits from `collections.UserString`. This was done to avoid confusion between `str` and `HTML` objects. (#86)
+
+* `Tag` attributes no longer silently support `HTML` as values. (#86)
+
+* `Tag` and `TagList`'s method `.get_html_string()` now both return `str` instead of `HTML`. (#86)
```

Comment:
After talking about this in person, I think either way is defendable (returning a `str` or `HTML`). Let's just make sure the type is consistent with `RenderedHTML`

Reply:
I think it makes sense that `RenderedHTML`'s `html` field should still be a `str`.

Reply:
Also, I agree with `.get_html_string()` returning `str`.